src/pseudopeople/configuration/noise_configuration.py

```diff
@@ -76,14 +77,45 @@ def get_value(
                 f"Available parameters are {list(parameter_tree.keys())}."
             )
         noise_value: Union[int, float, LayeredConfigTree] = parameter_tree[parameter_name]
-        # TODO: [MIC-5238] deal with properly when updating column noising, possibly with custom getter
         converted_noise_value: Union[int, float, dict] = (
             noise_value.to_dict()
             if isinstance(noise_value, LayeredConfigTree)
             else noise_value
         )
         return converted_noise_value
 
+    def set_value(
+        self,
+        dataset: str,
+        noise_type: str,
+        parameter_name: str,
+        new_value: Union[float, int, list, dict],
+        column_name: Optional[str] = None,
+    ) -> None:
+        if parameter_name == Keys.POSSIBLE_AGE_DIFFERENCES:
+            if isinstance(new_value, dict) or isinstance(new_value, list):
+                new_value = self.format_misreport_age_perturbations(dataset, new_value)
```

Comment:
This function duplicates some processing that would occur when passing in an override layer when creating a noise configuration object.